common/app/model/content.scala

```diff
@@ -389,7 +388,7 @@ object Content {
       showInRelated = apifields.flatMap(_.showInRelatedContent).getOrElse(false),
       cardStyle = CardStyle.make(cardStyle),
       shouldHideAdverts = apifields.flatMap(_.shouldHideAdverts).getOrElse(false),
-      shouldHideReaderRevenue = apifields.flatMap(_.shouldHideReaderRevenue).getOrElse(false),
+      shouldHideReaderRevenue = fields.shouldHideReaderRevenue.getOrElse(false),
```

Comment:
is this still necessary? if it's in Fields maybe we don't need it in Content?

Reply:
👍